ar71xx: Move F9K1115v2 under ar71xx tiny target

kchan345 opened the pull request:
F9K1115v2 has a kernel partition size of 1408 kB.

Since kernel 4.9.x+ the kernel image for this device compiled had exceeded
the kernel partition size limit and thus failing size check.

The kernel image generated for this device
under ar71xx tiny target is 1329.67 kB < 1408 kB.

Signed-off-by: Kin Chan <kcchan1@outlook.com>

blogic commented on target/linux/ar71xx/image/legacy.mk:
```diff
@@ -1011,6 +1009,8 @@ $(eval $(call SingleProfile,WHRHPG300N,64kraw,WLAEAG300N,wlae-ag300n,WLAE-AG300N
 
 $(eval $(call SingleProfile,ZyXEL,64k,NBG_460N_550N_550NH,nbg460n_550n_550nh,NBG460N,ttyS0,115200,NBG-460N))
 
+$(eval $(call SingleProfile,Belkin,64k,F9K1115V2,f9k1115v2,F9K1115V2,ttyS0,115200,$$(f9k1115v2_mtdlayout),BR-6679BAC))
```
incorrect alphabetical ordering

blogic commented on target/linux/ar71xx/image/tiny-legacy-devices.mk:
```diff
@@ -152,3 +152,10 @@ define LegacyDevice/NBG_460N_550N_550NH
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport \
+        kmod-ath10k ath10k-firmware-qca988x
+endef
+LEGACY_DEVICES += F9K1115V2
```
this is also not ordered correctly

kchan345 replied:
Hi,

I moved belkin to the top. Existing entries were not ordered alphabetically based on device name or device title. Let me know where you would like me to place the entry for f9k1115v2.

Commit:
ar71xx: Move F9K1115v2 under ar71xx tiny target

F9K1115v2 has a kernel partition size of 1408 kB.

Since kernel 4.9.x+ the kernel image for this device compiled had exceeded
the kernel partition size limit and thus failing size check.

The kernel image generated for this device
under ar71xx tiny target is 1329.67 kB < 1408 kB.

Signed-off-by: Kin Chan <kcchan1@outlook.com>

blogic commented:
merged, Thanks !
